src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -322,7 +325,8 @@ private AdminSearchPageData extractDataFromFeedbackSession(FeedbackSessionAttrib
                 data.studentOpenFeedbackSessionLinksMap.get(student.getIdentificationString()).add(submitUrl);
             }
 
-            data.feedbackSessionLinkToNameMap.put(submitUrl, fsa.getFeedbackSessionName());
+            data.feedbackSessionLinkToNameMap.put(submitUrl, fsa.getFeedbackSessionName()
+                    + generateOpenCloseDateInfo(fsa.getSessionStartTime(), fsa.getSessionEndTime()));
```

Comment:
`generateOpenCloseDateInfo(...)` can be done once at the start, instead of repeating it each time.